storybookjs/storybook
# add ability to have multiple sections in the notes panel

## What I did

A user can now specify multiple notes with `addon-notes`

![Screen Shot 2019-05-22 at 5 33 49 PM](https://user-images.githubusercontent.com/1192452/58217465-c8215d80-7cb7-11e9-939c-26e0493b1ba3.png)
![Screen Shot 2019-05-22 at 5 26 52 PM](https://user-images.githubusercontent.com/1192452/58217435-ad4ee900-7cb7-11e9-92af-4b655703668a.png)
![Screen Shot 2019-05-22 at 5 30 50 PM](https://user-images.githubusercontent.com/1192452/58217436-ad4ee900-7cb7-11e9-8295-e0f19ad659f7.png)

## How to test

- Is this testable with Jest or Chromatic screenshots? yes
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? yes

If your answer is yes to any of these, please make sure to include it in your PR.
